app/app-models.js

```diff
@@ -6,10 +6,22 @@ exports.selectTopics = () => {
 }
 
 exports.selectEndpoints = () => {
-    console.log(__dirname)
     return fs.readFile(`${__dirname}/../endpoints.json`, "utf-8").then((fileContents) => {
         const endpoints = JSON.parse(fileContents)
         return endpoints
     })
 }
 
+exports.findArticleById = (article_id) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = ${article_id}`).then((result) => {
+        const article = result.rows[0]
```

Comment:
Big safety issue here. You are opening yourself up for an SQL injection. This needs to be changed :)